grails-app/services/org/broadinstitute/orsp/webservice/ApiService.groovy

```diff
@@ -35,15 +35,8 @@ class ApiService {
         queryService.findByQueryOptions(options).collect{it.projectKey}.unique().sort()
     }
 
-    Collection<Map<String, Object>> getProjectSummaries(String term) {
-        QueryOptions options = new QueryOptions()
-        if (term) options.freeText = term
-        options.issueTypeNames.addAll(
-                [IssueType.IRB.name,
-                 IssueType.NHSR.name,
-                 IssueType.NE.name,
-                 IssueType.CONSENT_GROUP.name])
-        filterUnique(queryService.findByQueryOptions(options).collect {
+    Collection<Map<String, Object>> getProjectSummaries() {
```

Comment:
I'm pretty sure this method is eventually called from the swagger page (but can't check it for real, it's broken):
```
  /api/projects:
    get:
      tags:
        - Projects
      summary: Project Summaries
      description: Project Summaries
      parameters:
        - in: query
          name: term
          schema:
            type: string
          required: false
          description: Returns projects matching query term against all project fields
      responses:
        200:
          description: Projects
          schema:
            type: array
            items:
              $ref: '#/definitions/Project'
        500:
          description: Server Error
```
And since this method no longer takes a term, the swagger spec is not correct.

Reply:
Hi Greg, finally I've added the term functionality again, so swagger it's working now.